update.php

```diff
@@ -214,6 +214,7 @@
  */
 $phpunit_versions = array(
 	'9' => array(
+		'8.1',
 		'8.0',
 		'7.4',
 		'7.3',
```

Comment:
It looks like `8.0` is missing from the PHPUnit 8 array. Should we add `8.0` (and `8.1`?) to that list here as well.

Reply:
@desrosj Good question. I'm in two minds about it as support for PHP 8+ is not feature complete in PHPUnit 8.5. This might lead to unexpected results for people. On the other hand, if people choose to run PHPUnit 8 on PHP 8+, it's their responsibility to know its limitations.

What do you think ?

Reply:
I think it should be OK to publish the images for each combination.

Currently, the default version of PHPUnit on PHP 8.0 and 8.1 is 9. This is defined higher up in `$php_versions`. Adding PHP versions to the `$phpunit_versions` array will only result in `9-php-8.1-fpm`, `8-php-8.1-fom`, etc. containers being published. It would be a pretty intentional decision to use one of these specific containers instead of falling back to the default version for PHP 8.0 and 8.1.

It's also worth noting that the default PHPUnit container for PHP 8.0 has not worked since being first published about a year ago, so this wouldn't represent a breaking change. If folks are running tests on PHP 8 and still need to support older PHP/PHPUnit combinations, they're likely using a Composer based solution like Core has.

Reply:
Well, actually... running the tests on PHP 8.0 with the PHPUnit (8/9) Phar will now work again for Core, as long as the dependencies (PHPUnit Polyfills) are (Composer) installed.... ;-)

Reply:
Rebased the PR and added the extra versions in a separate commit.